ndinterp_capi/Cargo.toml

```diff
@@ -1,14 +1,21 @@
 [package]
 name = "ndinterp_capi"
 version = "0.0.1"
+authors = ["Alessandro Candido <alessandro.candido@sns.it>", "Juan Cruz-Martinez <juacrumar@lairen.eu>"]
```

Comment:
Here, if you put my name, add also @cschwan, we contributed at the same level. Otherwise, just put yours :)

Reply:
Btw, you can add @cschwan even in the main one, if he's okay with that.